readme.md

```diff
@@ -450,3 +461,36 @@ body, .foo {
   [RULE_END]  
 ]
 ```
+
+#### JavaScript Function Value
```

Comment:
> Also, it's highly unlikely that we'll have control over all function's values. We're implying here that all results of such interpolations need to be preparsed (think babel plugin), so in the case of SC every function return value would need to be wrapped in a css tagged template string helper so that it's targetable in a babel plugin.

I... think we can enforce that all interpolations return valid ISTF tokens:

```js
styled.div`
  box-shadow: 0 0 4px ${ props => props.shadowColor };
`
```

Obvs this interpolation causes problems, since it's returning only part of a `VALUE`. But, it could be rewritten as:

```
const PARTIAL_INTERP_1 = props => props.shadowColor
styled.div`
  box-shadow: ${ props => `0 0 4px ${PARTIAL_INTERP_1(props)}` };
`
```

We could build these functions up on the fly to handle any of these cases. For example `.${ OuterComponent } > & { ... }`

This might cause other issues with interop but I'm not sure just yet.